src/pybind/mgr/nfs/export.py

```diff
@@ -76,6 +77,16 @@ def validate_cephfs_path(mgr: 'Module', fs_name: str, path: str) -> None:
         raise NFSException(e.args[1], -e.args[0])
 
 
+def _validate_cmount_path(mgr: 'Module', fs_name: str, cmount_path: str, path: str) -> None:
+    validate_cephfs_path(mgr, fs_name, cmount_path)
+    if len(cmount_path) > len(path):
```

Comment:
Help me understand this a bit:
1) You validate `cmount_path` using the helper and then compare the length of it against `path` which is not validated? Doesn't this sound a bit of an assumption?
2) Comparing lengths of a path using `len` might work but I'm finding it a bit off-putting. I'd rather check if both the paths are correct and then just do `if cmount_path not in path: raise ValueError()`?
E.g:
- `cmount_path` = `/foo/bar` & `path` = `foo/bar/baz` # works fine
- `cmount_path` = `/foo/bar/baz/foo1` & `path` = `foo/bar/baz` # oops is not valid
Additionally you can put checks like
```
cmount_path = cmount_path.strip('/')
path = path.strip('/')
```
to make sure the forward slashes don't becomes a blocker

Reply:
Well path is already validated please check [here](https://github.com/ceph/ceph/pull/54277/files#diff-d00d146481b43490d2e6e79fcf061bf8fbd7364614e01613320d6e2234025c02R680)
So u can't compare any random values, here it's made sure that mentioned path is valid cephfs path or not which is done for both `path` & `cmount_path`. 
And well it's not actually  `if cmount_path not in path` cmount_path must be super set of path  which is what doc says so it's valid so called assumption here. Anyways I can do something like this instead  
`if path not in cmount_path:` if that doesn't seem too off-putting?